README.md

```diff
@@ -2,7 +2,16 @@
 
 This is the master index used by the [CentOS Community Container Pipeline Service](https://github.com/CentOS/container-pipeline-service) to build and deliver containers at [registry.centos.org](https://wiki.centos.org/ContainerPipeline)
 
-# Quickstart
+## Adding your project to index
+
+Any project is welcome to add their containers to the container index, so long as
+
+1. Project is opensource, and they are ok with their image being published publically on registry.centos.org, alongwith the logs, such as build logs, scan logs etc.
+2. Image is CentOS based (eg: dockerfile has ```FROM centos``` or its eventual base image).
```

Comment:
Suggestion:
Image is CentOS based, i.e., Dockerfile has ```FROM centos``` or its eventual base image.